imports/ui_3/views/common/page-heading.html

```diff
@@ -19,14 +19,5 @@ <h2>{{title}}</h2>
             </ol>
             {{/unless}}
         </div>
-        {{#if printable}}
```

Comment:
A szamla email/ben adunk egy linket hogy "Nyomtathato nezet" es az erre a szamla page-re mutat. Szoval nem vehetjuk le rola a nyomtatas gombot. Ha ezt adjuk mint nyomtathato nezet.
Az tok jo hogy a modalon is van mar Nyomtatas gomb, de a Moda-ra meg linket adni nem lehet.

Reply:
Csak a headerbol vettem ki a nyomtatas gombot, mert teljesen mashogy nezett ki mint mashol, meg a helye sem volt tul logikus. A szamlan ott a gomb.